### PR TITLE
Fix 2885 & 2905

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by antgonza** [comment]
Forgot to add:

**Comment by antgonza** [comment]
By mistake pushed to this same branch the fix for: https://github.com/biocore/qiita/issues/2886. See new functionality: 
uk

File: qiita_pet/templates/study_ajax/sample_prep_summary.html [hunk 1]
```diff
@@ -247,9 +269,12 @@
     <tr>
       <td>
         <h3>Sample Summary</h3><br/>
-        <button class="btn btn-danger st-interactive" onclick="sampleTemplatePage.$refs.stElem.deleteSamples($('.sample-delete:checkbox:checked'));">
+        <button class="btn btn-danger st-interactive" onclick="deleteIndividualSamples($('.sample-delete:checkbox:checked'))">
             <span class="glyphicon glyphicon-trash"></span> Delete Selected
         </button>
+        <button class="btn btn-danger st-interactive" onclick="deleteNotOverlappingSamplePrepSamples()">
+            <span class="glyphicon glyphicon-trash"></span> Delete Not Overlapping Samples
```
Comment (charles-cowart) [inline qiita_pet/templates/study_ajax/sample_prep_summary.html]: 'Not Overlapping' might be better expressed as 'non-overlapping'.

File: qiita_pet/templates/study_ajax/sample_prep_summary.html [hunk 2]
```diff
@@ -28,6 +30,26 @@
...
   }
+  function deleteIndividualSamples(samples){
+    var sample_names = [];
+    samples.each(function(){
+      sample_names.push($(this).prop('name'));
+    });
+    sampleTemplatePage.$refs.stElem.deleteSamples(sample_names);
+  }
+  function deleteNotOverlappingSamplePrepSamples(){
+    if (confirm('Are you sure you want to delete all samples that have _not_ being linked to a preparation?')){
```
Comment (charles-cowart) [inline qiita_pet/templates/study_ajax/sample_prep_summary.html]: being -> been

**Pull request merged**